### ISSUE TITLE
translation error on product position placement page

### ISSUE THREAD

**Issue by alfonsolr09** [PrestaShop/PrestaShop#28211]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

the translation of the button is wrong, it should say something like "reorder" ("reordenar" in Spanish)

![image](https://user-images.githubusercontent.com/62474824/162597851-9d8494c8-b93d-4f46-a77f-6a7b74869e2d.png)

### Steps to reproduce

1. Bo
2. Catalog, products
3. Try to reorder any product (have Spanish as a language)

### PrestaShop version(s) where the bug happened

1.7.8.5

### PHP version(s) where the bug happened

7.3

**Comment by alfonsolr09** [comment]
![image](https://user-images.githubusercontent.com/62474824/162598011-04541239-820b-4b0c-971e-1313d44c7457.png)

**Comment by hibatallahAouadni** [comment]
Hello @alfonsolr09 

I reproduce the issue with PS1778, PS1785 and develop, see the attached screenshot below:

![image](https://user-images.githubusercontent.com/23308427/162710150-2072d689-b90b-4bdc-9c94-dfe13ea7d27b.png)

I’ll add this to the debug backlog so that it’s fixed.

Please be aware that due to the high number of reported issues, some bugs might take a very long time to be resolved. If this issue is important to you and you cannot wait for it to be fixed on the project’s own time, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you. 

If you manage to get the problem fixed, then please contribute the fix back to the project as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/). Remember that the more people contribute, the better PrestaShop becomes for everyone.

Ping @Julievrz wdyt of this issue, should it be fixed on Core or in Crowdin, thanks in advance :pray: 

Thank you

**Comment by Julievrz** [comment]
Hello, this should be fixed in Crowdin. An issue was already opened here: https://crowdin.com/translate/prestashop-official/36366/en-es?filter=basic&value=0#5845406. I've asked the Spanish proofreaders to review it. :-)

**Comment by alfonsolr09** [comment]
> Hello, this should be fixed in Crowdin. An issue was already opened here: https://crowdin.com/translate/prestashop-official/36366/en-es?filter=basic&value=0#5845406. I've asked the Spanish proofreaders to review it. :-) 
> 
> 

yes, I did it and I corrected it. will it merge automatically? can i close this issue?

**Comment by Julievrz** [comment]
> > Hello, this should be fixed in Crowdin. An issue was already opened here: https://crowdin.com/translate/prestashop-official/36366/en-es?filter=basic&value=0#5845406. I've asked the Spanish proofreaders to review it. :-)
> 
> yes, I did it and I corrected it. will it merge automatically? can i close this issue?

A proofreader validated it. It should take a couple of days before it's available on PrestaShop software. Yes, this issue can be closed. ☺️

**Comment by alfonsolr09** [comment]
@Julievrz the translations are updated with the next versions of PrestaShop?? (1.7.8.6) or should I update it manually?

**Comment by alfonsolr09** [comment]
@Julievrz the translations are updated with the next versions of PrestaShop?? (1.7.8.6) or should I update it manually?

**Comment by Julievrz** [comment]
> @Julievrz the translations are updated with the next versions of PrestaShop?? (1.7.8.6) or should I update it manually?

You need to go to International > Translations > Add/Update a language > Select your language and click "Update". :-) It will take a couple of days.

**Comment by alfonsolr09** [comment]
> > @Julievrz the translations are updated with the next versions of PrestaShop?? (1.7.8.6) or should I update it manually?
> 
> You need to go to International > Translations > Add/Update a language > Select your language and click "Update". :-) It will take a couple of days.

still not working, not updated yet

**Comment by alfonsolr09** [comment]
@Julievrz ??

**Comment by alfonsolr09** [comment]
![image](https://user-images.githubusercontent.com/62474824/176542485-380e8353-c6b9-4bc5-b07a-d47a5909b3cd.png)

same problem in 8.0

**Comment by Julievrz** [comment]
> @Julievrz ??

The string was validated by a proofreader a while ago so it should work. @PrestaShop/qa-team Could someone double-check, please?

**Comment by hibatallahAouadni** [comment]
Hello @Julievrz 

I still reproduce the issue with PS8.0.0, see the attached screenshot below:

![image](https://user-images.githubusercontent.com/23308427/176650765-df9aaf4a-13a5-4e49-a37e-50e35ed0b68a.png)

Thanks!

**Comment by Julievrz** [comment]
@hibatallahAouadni I'm afraid there's nothing more I can do on Crowdin. I think we should investigate the issue further.

**Comment by Hlavtox** [comment]
Closed, not relevant anymore.
